src/Components/AuthBackground/AuthBackground.tsx

```diff
@@ -13,10 +13,35 @@ const AuthBackground = () => {
       : pathname === "reset-password"
       ? "auth-background3"
       : "auth-background";
+  const sectionTitle =
```

Comment:
You can abstract this logic into a function that will take the pathname as a parameter and returns background Image, section description, section title.